src/latex/references.bib

```diff
@@ -684,3 +684,50 @@ @misc{small-step-semantics
   url   = {https://github.com/intersectmbo/cardano-ledger/releases/latest/download/small-step-semantics.pdf},
 }
 
+@online{www-developers-cardano-reintro,
+  author = {Eduardo Garrido and the Cardano community},
+  title = {(Re)introduction to Cardano},
+  year = 2024,
+  url = {https://developers.cardano.org/docs/operate-a-stake-pool/introduction-to-cardano/},
+  urldate = {2025-03-12}
+}
+@online{www-docs-cardano-network,
+  url = {https://docs.cardano.org/about-cardano/explore-more/cardano-network},
+  year = {2025},
+  title = {{C}ardano network},
+  urldate = {2025-03-12}
+}
+@online{www-docs-cardano-time,
+  url = {https://docs.cardano.org/about-cardano/explore-more/time},
+  year = {2025},
+  title = {{T}ime handling on {C}ardano},
+  urldate = {2025-03-12}
+}
+@online{www-essential-cardano-glossary,
+  url = {https://www.essentialcardano.io/glossary?sort=alphabetical},
+  year = {2025},
+  title = {Glossary},
+  urldate = {2025-03-12}
+}
+@online{www-iohk-blog-ouroboros,
+  author = {Olga Hryniuk},
+  url = {https://iohk.io/en/blog/posts/2021/10/27/ouroboros-chronos-provides-the-first-high-resilience-cryptographic-time-source-based-on-blockchain/},
+  year = {2021},
+  title = {Ouroboros Chronos provides the first high-resilience, cryptographic time source based on blockchain technology},
+  urldate = {2025-03-12}
+}
+@online{www-ledger-academy-glossary,
+  title = {Crypto Glossary},
+  url = {https://www.ledger.com/academy/glossary},
+  year = {2024},
+  urldate = {2025-03-12}
+}
+@online{www-ledger-academy-stake-ada,
+  title = {{C}ardano {S}taking: {H}ow {T}o {S}take {ADA}},
+  url = {https://www.ledger.com/academy/cardano-staking-how-to-stake-ada},
+  year = {2024},
+  month = {January},
+  day = {5},
+  urldate = {2025-03-12}
+}
```

Comment:
In retrospect, perhaps we don't want online resources in the References section and should just insert href (links) in the document instead.

Reply:
@carlostome wdyt?